pkg/local_object_storage/writecache/delete.go

```diff
@@ -27,17 +27,22 @@ func (c *cache) Delete(addr *objectSDK.Address) error {
 	c.mtx.Unlock()
 
 	// Check disk cache.
-	has := false
+	var has []byte
 	_ = c.db.View(func(tx *bbolt.Tx) error {
 		b := tx.Bucket(defaultBucket)
-		has = b.Get([]byte(saddr)) != nil
+		has = b.Get([]byte(saddr))
```

Comment:
The returned value of `Get` is only valid during tx execution. We need to remember it's length instead (`var has bool` -> `var sz int`) and compare it with 0.

Reply:
Done.